nwb_explorer/nwb_model_interpreter/nwb_reader.py

```diff
@@ -24,7 +26,10 @@ def get_plottable_timeseries(time_series, resampling_size=None):
 
         # TODO we may need to rearrange that when dealing with spatial series: a different type of plot (3D or else)
         #  may me more adequate than what we're doing (i.e. splitting in multiple mono dimensional timeseries)
-        time_series_array = NWBReader.get_mono_dimensional_timeseries_aux(time_series.data[::1])
+        d = time_series.data[::1]
```

Comment:
@pgleeson related? https://github.com/MetaCell/nwb-explorer/issues/63

Reply:
Yes, related, but the curent modification would probably only be relevant for a 1D spatial time series... and yes ideally you'd like to be able to plot that in a standard graph, e.g. position of mouse moving in 1D.

The 2D option would best be displayed in NWBE as a 2D grid with a moving point...